buildSrc/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'java'
+
+repositories {
+    mavenLocal()
```

Comment:
Maven local can be removed (only useful if you have gradle + maven builds in your system), will speed up the build.